src/components/Blocks/BlockDetail.jsx

```diff
@@ -3,44 +3,46 @@ import { useParams, Link } from 'react-router-dom'
 
 import { UserContext } from '../User/handleUser'
 
-import '../../common.css'
+// import '../../common.css'
```

Comment:
please remove this and delete the file if not needed

Reply:
done